graviti/schema/catalog_to_schema.py

```diff
@@ -0,0 +1,322 @@
+        super().__init__(label_type)
+        self._portex_type = portex_type
+        self.import_types = {"name": portex_type}
+
+    def __call__(  # type: ignore[override]
+        self, subcatalog: SubcatalogBase, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        raise NotImplementedError
+
+
+class _ConverterSingleLabel(_ConverterCommon):  # pylint: disable=too-few-public-methods
+    def __init__(self, label_type: str, portex_type: str) -> None:
+        super().__init__(label_type, portex_type)
+        self._field_name = label_type
+
+    def __call__(  # type: ignore[override]
+        self, subcatalog: SubcatalogBase, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        return {
+            "name": self._field_name,
+            "type": self._portex_type,
+            **_get_category_and_attribute_params(subcatalog),
+        }
+
+
+class _ConverterInstanceMask(_ConverterSingleLabel):  # pylint: disable=too-few-public-methods
+    def __call__(  # type: ignore[override]
+        self, subcatalog: InstanceMaskSubcatalog, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        field = super().__call__(subcatalog, labels)
+        is_tracking = subcatalog.is_tracking
+        if is_tracking:
+            field["isTracking"] = is_tracking
+        return field
+
+
+class _ConverterPanopticMask(_ConverterSingleLabel):  # pylint: disable=too-few-public-methods
+    def __call__(  # type: ignore[override]
+        self, subcatalog: PanopticMaskSubcatalog, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        field = super().__call__(subcatalog, labels)
+        field["stuffIds"] = field.pop("categoryIds")
+        return field
+
+
+class _ConverterMultiLabel(_ConverterCommon):  # pylint: disable=too-few-public-methods
+    def __init__(self, label_type: str, portex_type: str) -> None:
+        super().__init__(label_type, portex_type)
+        self._field_name = label_type.upper() if label_type == "rle" else f"{label_type}s"
+
+    def __call__(  # type: ignore[override]
+        self, subcatalog: SubcatalogBase, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        items = {"type": self._portex_type, **_get_category_and_attribute_params(subcatalog)}
+        return {"name": self._field_name, "type": "array", "items": items}
+
+
+class _ConverterMultiGeometryLabel(_ConverterMultiLabel):  # pylint: disable=too-few-public-methods
+    def __init__(self, label_type: str, portex_type: str, label_value_level: int) -> None:
+        super().__init__(label_type, portex_type)
+        self._label_value_level = label_value_level
+
+    def __call__(  # type: ignore[override]
+        self, subcatalog: SubcatalogBase, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        field = super().__call__(subcatalog, labels)
+
+        if labels:
+            value = labels
+            for _ in range(self._label_value_level):
+                value = value[0]  # type: ignore[assignment]
```

Comment:
why we ignore this?

Reply:
Because the type of `value` is different from its initial type.